### ISSUE TITLE
Typescript error when importing AriaType

### ISSUE THREAD

**Issue by ischemicbg** [ddddurk/ariatype#63]
I'm getting the following error in an Angular 20 app when trying to import `AriaType`

`X [ERROR] TS2307: Cannot find module '@ariatype/aria-attributes/dist' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/@ariatype/ariatype/dist/index.d.ts:15:31:
      15 │ import { AriaAttributes } from '@ariatype/aria-attributes/dist';
         ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] TS2307: Cannot find module '@ariatype/aria-roles/dist' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/@ariatype/ariatype/dist/index.d.ts:16:26:
      16 │ import { AriaRoles } from '@ariatype/aria-roles/dist';
         ╵                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] TS2307: Cannot find module '@ariatype/partially-required-props/dist' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/@ariatype/ariatype/dist/index.d.ts:17:39:
      17 │ ...llyRequiredProps } from '@ariatype/partially-required-props/dist';
`

If I go into the index.d.ts and remove the `dist` from each of the failing import paths it works fine but that isn't very practical

**Comment by durksteedjr** [comment]
Hello! Please upgrade to `@ariatype/ariatype` version `1.0.2`, hopefully this will be fixed. Thank you!

**Comment by ischemicbg** [comment]
Thanks, That did indeed solve the issue.
